src/main/resources/db/migration/local+dev/20220913131811__add_ap_service_name_to_mock_premises.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE premises
```

Comment:
Can you change the name of the migration so the timestmap is accurate to today?  e.g.

`20221101124300__add_ap_service_name_to_mock_premises.sql`

the PR description here goes over why: https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/197